### DISCUSSION TITLE
Wrong unit (öre) in card_nordpool_energy_chart.yaml

### DISCUSSION THREAD

**Discussion by Wictorssons** [TypQxQ/Sigenergy-Home-Assistant-Integration#18] [General]
The chart shows "öre" and "öre/kWh" when the code display SEK.
I changed it to "SEK" and "SEK/kWh" and the number of y-axis digits to "2" instead of "0" to get it to display correctly.
I actually prefer SEK instead of öre, so for me this setup works best. Thanks for a nice graph!

**Comment by TypQxQ** [comment]
Hi! Not sure I understand what you mean.
The nordpool_energy_chart is hardcoded to öre and you need to manually change to the currency you like and want to use together with the Nordpol integration and with the calculations done inside the card.

**Reply by Wictorssons** [reply]
I guess I was just lazy and expected the mapping of the currency to just work. By default my Nordpool sensor is using SEK but of course it is up to the user which sensor to use. It was an easy fix to adapt it to the one I use. I understand that it should not be considered an error or bug, so you can close this conversation. :-)
